pkgs/misc/betterfox/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "betterfox";
+  version = "119.0";
+
+  src = fetchFromGitHub {
+    owner = "yokoffing";
+    repo = "Betterfox";
+    rev = version;
+    hash = "sha256-FX5y7fmxTcyB5aVb1IDOwhZPRB/VyqPaIVabMuiOrv8=";
+  };
+
+  dontPatch = true;
+  dontConfigure = true;
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/share/betterfox
+    cp -r * $out/share/betterfox/
+  '';
+
+  meta = with lib; {
+    description = "Firefox speed, privacy, and security: a user.js template for configuration. Your favorite browser, but better.";
```

I think this describes what the package actually is a bit better.

```suggestion
    description = "A modern opinionated user.js template for Firefox";
```